cmd/neofs-cli/modules/util/acl.go

```diff
@@ -278,23 +289,52 @@ func parseEACLRecord(args []string) (*eacl.Record, error) {
 func parseKVWithOp(s string) (string, string, eacl.Match, error) {
 	i := strings.Index(s, "=")
 	if i < 0 {
+		if i = strings.Index(s, "<"); i >= 0 {
+			if !validateDecimal(s[i+1:]) {
+				return "", "", 0, fmt.Errorf("invalid base-10 integer value %q for attribute %q", s[i+1:], s[:i])
+			}
+			return s[:i], s[i+1:], eacl.MatchNumLT, nil
+		} else if i = strings.Index(s, ">"); i >= 0 {
+			if !validateDecimal(s[i+1:]) {
+				return "", "", 0, fmt.Errorf("invalid base-10 integer value %q for attribute %q", s[i+1:], s[:i])
+			}
+			return s[:i], s[i+1:], eacl.MatchNumGT, nil
+		}
+
 		return "", "", 0, errors.New("missing op")
 	}
 
-	var key, value string
-	var op eacl.Match
+	if len(s[i+1:]) == 0 {
+		return s[:i], "", eacl.MatchNotPresent, nil
+	}
+
+	value := s[i+1:]
 
-	if 0 < i && s[i-1] == '!' {
-		key = s[:i-1]
-		op = eacl.MatchStringNotEqual
-	} else {
-		key = s[:i]
-		op = eacl.MatchStringEqual
+	if i == 0 {
+		return "", value, eacl.MatchStringEqual, nil
 	}
 
-	value = s[i+1:]
+	switch s[i-1] {
```

Comment:
Some proper parser would be handy here, but luckily we don't have a lot of options now.